web/index.html

```diff
@@ -0,0 +1,80 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8"/>
+    <title>Zone Builder demo</title>
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.7.1/dist/leaflet.css" integrity="sha512-xodZBNTC5n17Xt2atTPuE1HxjVMSvLVW9ocqUKLsCC5CXdbqCmblAshOMAS6/keqq/sMZMZ19scR4PsZChSR7A==" crossorigin=""/>
+    <script src="https://unpkg.com/leaflet@1.7.1/dist/leaflet.js" integrity="sha512-XQoYMqMTK8LvdxXYG3nZ448hOEQiglfqkJs1NOQV44cWnUrBc8PkAOcXy20w0vlaXaVUearIOBhiXZ5V3ynxwA==" crossorigin=""></script>
+    <style>
+        body {
+            margin: 0px;
+            border: 0px;
+            padding: 0px;
+        }
+
+        #map {
+            height: 90%;
+            width: 90%;
+            position: absolute;
+        }
+    </style>
+  </head>
+  <body>
+	  Circles: <input type="range" min="1" max="10" value="5" id="num_circles">
+	  Segments: <input type="range" min="1" max="20" value="23" id="num_segments">
+	  <button type="button" onclick="window.regenerate();">Generate</button>
+    <div id="map"></div>
+    <script type="module">
+        import init, {make_clockboard} from "./pkg/zonebuilder.js";
+
+	// Initialize the WASM function.
+	// TODO To be safe, we should really wait for it to complete before proceeding.
+        init();
+
+	// Create the Leaflet map
```

Comment:
I pieced this together from the excellent https://leafletjs.com/examples.html tutorials